src/models/transfers.js

```diff
@@ -434,10 +447,7 @@ function * setTransfer (transfer, requestingUser) {
   }
 }
 
-function * getFulfillment (transferId, requestingUser) {
-  if (!requestingUser || !requestingUser.name) {
```

Comment:
Since getting the transfer is unrestricted, getting the fulfillment should be as well for consistency. I think you can make a stronger argument for making the transfer details private than making the fulfillment private, since the fulfillment doesn't actually contain any meaningful information about the transfer.

So my thought was to remove the authentication from all transfer endpoints for now (simplicity/consistency) and then consider whether we want to add it to getting/putting transfers separately.